### ISSUE TITLE
Add rule to disallow alternative control structures

### ISSUE THREAD

**Issue by maxolasersquad** [squizlabs/PHP_CodeSniffer#3413]
It would be helpful to have a rule that would disallow the [alternative control structures](https://www.php.net/manual/en/control-structures.alternative-syntax.php) for PHP. We do not allow them in our code, but some contractors keep using them. It would be nice for phpcs to complain about them before we get to code review.

**Comment by jrfnl** [comment]
@maxolasersquad I'm happy to be able to tell you such a rule already exists, though it is in an external standard - PHPCSExtra:
https://github.com/phpcsstandards/phpcsextra#universalcontrolstructuresdisallowalternativesyntax-wrench-bar_chart-books

**Comment by maxolasersquad** [comment]
```xml
    <rule ref="Universal.ControlStructures.DisallowAlternativeSyntax">
        <properties>
            <property name="allowWithinInlineHTML" value="false"/>
        </properties>
    </rule>
```
Yep. Worked like a champ. Thank you.
